cw_bitcoin/lib/electrum_wallet.dart

```diff
@@ -1444,6 +1450,9 @@ abstract class ElectrumWalletBase
         updatedUnspentCoins.add(coin);
       } catch (_) {}
     }));
+    } catch (_) {
+      rethrow;
+    }
```

Comment:
I don't think we should rethrow while it's not handled correctly, and will just show the `send error report` popup to the user